user-service/controller/user-controller.js

```diff
@@ -1,6 +1,8 @@
-import { ormCreateUser as _createUser , ormFindUserbyUsername as _FindUserbyUsername} from '../model/user-orm.js'
-import UserModel from '../model/user-model.js';
-
+import { ormCreateUser as _createUser , 
+    ormFindUserbyUsername as _FindUserbyUsername, 
+    ormFindOneUser as _FindOneuser} from '../model/user-orm.js'
+import jwt from 'jsonwebtoken'
+import { ACCESS_TOKENT_SECRET } from '../constants/constants.js';

```

Comment:
inconsistent use of ; at the end of line
Present in multiple files, please standardize

Reply:
fixed